docs/components/tpr-header-bar.md

```diff
@@ -80,6 +87,34 @@ TPR Header Search implements the [alphagov/accessible-autocomplete](https://gith
 
 Views which require the `<tpr-header-search>` should also include the 'TPRHeaderSearchAutocomplete' partial view, in order to use the autocomplete functionality.
 
+### `<tpr-mobile-menu>`
```

Comment:
`TPRHeaderLockup.cshtml` uses `<tpr-header-menu>` so I think this documentation may be out-of-date.